src/main/scala/se/nimsa/sbx/metadata/MetaDataServiceActor.scala

```diff
@@ -60,6 +60,18 @@ class MetaDataServiceActor(metaDataDao: MetaDataDAO, propertiesDao: PropertiesDA
         case GetSeriesTags =>
           pipe(propertiesDao.listSeriesTags.map(SeriesTags)).to(sender)
 
+        case GetSeriesTag(tagId) =>
+          pipe(propertiesDao.seriesTagForId(tagId)).to(sender)
+
+        case UpdateSeriesTag(tag) =>
```

Comment:
The current pattern in slicebox for protecting the database from concurrent updates is to use `pipeSequentiallyTo` for anything that changes the database (update, create and delete here). Mix in the trait `SequentialPipeToSupport` to get this functionality.

Reply:
fixed